src/main/java/htsjdk/variant/vcf/AbstractVCFCodec.java

```diff
@@ -332,12 +332,12 @@ else if ( parts[2].equals(VCFConstants.EMPTY_ID_FIELD) )
         else
             builder.id(parts[2]);
 
-        final String ref = getCachedString(parts[3].toUpperCase());
-        final String alts = getCachedString(parts[4]);
+        final String ref = parts[3].toUpperCase();
+        final String alts = parts[4];
         builder.log10PError(parseQual(parts[5]));
 
         final List<String> filters = parseFilters(getCachedString(parts[6]));
-        if ( filters != null ) builder.filters(new HashSet<String>(filters));
+        if ( filters != null ) builder.filters(new HashSet<>(filters));
```

Comment:
may as well format the `if` across lines and use `{` `}` if you're changing this line

Reply:
done